chain-test/src/test/scala/org/bitcoins/chain/pow/BitcoinPowTest.scala

```diff
@@ -50,6 +50,25 @@ class BitcoinPowTest extends ChainUnitTest {
   }
 
   it must "calculate a GetNextWorkRequired correctly" taggedAs FixtureTag.PopulatedBlockHeaderDAO inFixtured {
-    case ChainFixture.PopulatedBlockHeaderDAO(blockHeaderDAO) => succeed
+    case ChainFixture.PopulatedBlockHeaderDAO(blockHeaderDAO) =>
+      val firstAfterAdjustmentF = blockHeaderDAO.getAtHeight(562464)
```

Comment:
What i had in mind here was iterating through each of the block's and verify that `getNextWorkRequired()` produces the correct output for the block that we have created in the fixture